rest/route/host.go

```diff
@@ -413,6 +413,13 @@ func (ch *offboardUserHandler) Run(ctx context.Context) gimlet.Responder {
 			"context": "user offboarding",
 			"user":    ch.user,
 		}))
+		err = user.ClearUserSettings(ch.user)
+		catcher.Wrapf(err, "clearing user settings for user '%s'", ch.user)
+		grip.Error(message.WrapError(err, message.Fields{
```

Comment:
I can't comment on the line because it's not modififed, but rather than logging here, can we just depend on the log on L427 (and change the error message about "not all unexpirable hosts/volumes terminated")?

Reply:
done

Reply:
Re 
`LGTM mod changing the error message string on [L423](https://github.com/evergreen-ci/evergreen/pull/5820/commits/2fefef22c21d3964ff34dd2fd3791347f2c69da4#diff-d3ff34a4c5e4308b36ed70a61b69676caec5ca99594eff8c800491065be4f594R423).`

I thought that was already done. Is this still outstanding?

Reply:
The "not all unexpirable hosts/volumes terminated" message is misleading because it could be due to terminating hosts/volumes or due to clearing user settings.

Reply:
GitHub doesn't let me link to a line that's not part of the diff: https://github.com/evergreen-ci/evergreen/blob/a4c509ab4daeddd20465bcebf57aebe6a355a7f4/rest/route/host.go#L421

Reply:
oh! of course! my bad, thank you.